includes/class-sensei-lesson.php

```diff
@@ -2658,70 +2658,6 @@ public function lesson_update_question_order_random() {
 		die();
 	}
 
-	/**
-	 * Save lesson course.
-	 *
-	 * @access private
-	 * @deprecated 2.2.0
-	 * @param array $data (default: array()).
-	 * @return integer|boolean $course_id or false
-	 */
-	private function lesson_save_course( $data = array() ) {
```

Comment:
I think we also have to put back this method and add it to the `4.0` issue because it's a dependency of the `lesson_add_course` :)

Reply:
Whoops! Moved that one too. 0ffb5b0